docs/sftt/sftt-api-spec.md

```diff
@@ -294,11 +345,11 @@ Invite someone to join a team. Will send an email to all specified people that i
 
 ##### `200 OK`
 
-!!! missing "Unknown"
+Users invited.
 
 ##### `400 BAD REQUEST`
 
-!!! missing "Should be an error here if the team id is invalid or if the user is already on the team"
+If the team id is invalid or if the user is already on the team.
```

Comment:
If any of the users is already on the team? Will the other users in the request still be invited? We have to specify exactly what is expected to happen.